tracer/build/_build/Build.Steps.cs

```diff
@@ -583,15 +584,16 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             CompressZip(SymbolsDirectory, WindowsSymbolsZip, fileMode: FileMode.Create);
         });
 
-    Target ZipTracerHome => _ => _
+    Target ZipMonitoringHome => _ => _
         .Unlisted()
-        .After(BuildTracerHome)
+        .After(BuildTracerHome, BuildProfilerHome, BuildNativeLoader)
         .Requires(() => Version)
         .Executes(() =>
         {
             if (IsWin)
             {
                 CompressZip(TracerHomeDirectory, WindowsTracerHomeZip, fileMode: FileMode.Create);
+                // for now we do not need a monitoring-home.zip file. So no need to create it.
```

Comment:
Well, I think arguably we _should_ have it, it's an alternative deployment mechanism (and it's how we run in AAS, so we probably should be doing the same thing there as elsewhere). 

I don't think we will ever have `monitoring-home.zip` though - it will need to remain being called `tracer-home.zip` for back-compat reasons

Reply:
For me this is linked to the discussion with the `windows-tracer-home.zip` and should be addressed later.